### DISCUSSION TITLE
Why doesn't airodump-ng restore packet capture sessions or retain the original time frames when capturing packets from a file?

### DISCUSSION THREAD

**Discussion by alexvillalb159** [aircrack-ng/aircrack-ng#2544] [General]
Dear Friends:

I've been using the airodump-ng tool, version 1.6, and I'm amazed at how powerful it is. But there is a behavior that puzzles me. When I try to rebuild all the csv files that by default are generated by airodump-ng when it takes input from a file, not from a wireless interface, it does not reproduce the temporary information of these. Packets that originally arrived with very long time intervals, it places them as they would have arrived in the same second.
The command I use is:

  airodump-ng -r airodump/all/all.out-01.cap -w /tmp/all.out

I understand that if I use :
  airodump-ng -T -r airodump/all/all.out-01.cap -w /tmp/all.out

then the time margins in the csv will be respected. But at the cost of having to wait for the packet to be copied at the same interval, it takes longer to be captured in real time via Wi-Fi. This is undesirable for me since I don't care to simulate it.

The reason why I want to do that is that I need to capture the data at different times or from different devices at the same time via wife and then merge all of that into a single file with the command:

merge cap

and then finally reproduce that information in the csv with the actual temporary information with which the files were generated separately. But when doing it with the default options of airodump-ng I lose all the temporary information that interests me.

Is this normal? Is there a way to do what I want? I'm surprised that airodump-ng doesn't even allow you to restore a packet capture session so you don't have to start from scratch.

In any case, thanks for this great tool.

**Comment by Mister-X-** [comment]
Not possible currently. You'd have to modify the code to do that. PR welcome.
